app/assets/javascripts/select-all.js

```diff
@@ -2,17 +2,25 @@
   "use strict";
 
   Modules.SelectAll = function () {
-    var that = this;
-
-    that.start = function (element) {
+    this.start = function (element) {
       function addEventListener() {
         var selectAll = element.find('#select_all');
+        var selectAllPages = element.find('#select_all_pages');
+        var selectAllPagesContainer = selectAllPages.closest('.if-js-hide');
 
         if (selectAll != undefined) {
           selectAll.on('change', function (e) {
             var checked = selectAll.prop('checked');
             var checkboxes = $('.select-content-item');
             checkboxes.prop('checked', checked);
+
+            if (selectAllPages != undefined) {
```

Comment:
Is this _ever_`false`? I thought jQuery's [`.find()`](https://api.jquery.com/find/) _always_ returned an [`jQuery`](https://api.jquery.com/Types/#jQuery) object? Shouldn't you check `if (selectAllPages.length)` ?